### DISCUSSION TITLE
Tabs not showing on qfield

### DISCUSSION THREAD

**Discussion by luwerli** [opengisch/QField#3032] [Bugs]
Hello,
I'm having a trouble with the use of qfield on a tablet or a phone. It seems that on QGIS my form works perfectly fine, with all the hidden tabs that it has in it, but when i import, both to the tablet and the phone, the tabs just don't show when i choose an option on a question that should reveal the tab.
If someone was experiencing the same thing and solved it, please let me know because right now i don't what else to do.

Luwerli

QGIS
![image](https://user-images.githubusercontent.com/86632260/176737317-cb3e8efb-88b3-4a34-a762-14469710c11d.png)
![image](https://user-images.githubusercontent.com/86632260/176737214-892b5b67-891d-4172-abb3-04e0329f4040.png)

QFIELD
![image](https://user-images.githubusercontent.com/86632260/176737881-598cf8ed-58b0-49fb-95e1-2ac3c56e2e5a.png)
![image](https://user-images.githubusercontent.com/86632260/176737915-04fc3cc0-b5be-480f-91ae-c786631fd64f.png)

**Comment by OBVYamaska** [comment]
I use these often with no issue. However, I have come across some problems over time. They almost always imply either a bad condition, or the screen management of your device!

Could you show the condition you're using to trigger the action?
